optimade/filtertransformers/mongo.py

```diff
@@ -201,27 +200,30 @@ def fuzzy_string_op_rhs(self, arg):
 
         if arg[0] == "CONTAINS":
             return {"$regex": f"{pattern}"}
-        elif arg[0] == "STARTS":
+        if arg[0] == "STARTS":
             return {"$regex": f"^{pattern}"}
-        elif arg[0] == "ENDS":
+        if arg[0] == "ENDS":
             return {"$regex": f"{pattern}$"}
 
+        raise TransformerError(
+            f'"fuzzy_string_op_rhs" can only have arguments "CONTAINS", "STARTS", and "ENDS". Passed: {arg[0]}'
+        )
+
     def set_op_rhs(self, arg):
         # set_op_rhs: HAS ( [ OPERATOR ] value | ALL value_list | ANY value_list | ONLY value_list )
 
         if len(arg) == 2:
             # only value without OPERATOR
             return {"$in": arg[1:]}
-        else:
-            if arg[1] == "ALL":
-                raise NotImplementedError
-            elif arg[1] == "ANY":
-                raise NotImplementedError
-            elif arg[1] == "ONLY":
-                raise NotImplementedError
-            else:
-                # value with OPERATOR
-                raise NotImplementedError
+
+        if arg[1] == "ALL":
+            raise NotImplementedError
+        if arg[1] == "ANY":
```

Comment:
I think `elif` is more readable but this is just a personal opinion. I have no hard feeling about it :)